P/PROJ/build_tarballs.jl

```diff
@@ -0,0 +1,87 @@
+using BinaryBuilder
+
+name = "PROJ"
+version = v"6.2.1"
+
+# Collection of sources required to build PROJ
+sources = [
+    "https://download.osgeo.org/proj/proj-$version.tar.gz" =>
+    "7f2e0fe63312f1e766057cceb53dc9585c4a335ff6641de45696dbd40d17c340",
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/proj-*/
+
+# sqlite needed to build proj.db, so this should not be the
+# cross-compiled one since it needs to be executed on the host
+apk add sqlite
+
+if [[ ${target} == *mingw* ]]; then
+    SQLITE3_LIBRARY=$prefix/bin/libsqlite3-0.dll
+elif [[ ${target} == *darwin* ]]; then
+    SQLITE3_LIBRARY=$prefix/lib/libsqlite3.dylib
```

Comment:
Use `${dlext}` as extension, so that you can cram all Unix systems together. For the directory, you can use `${libdir}` everywhere

Reply:
Ok, that's better indeed. I assumed by

> you can use ${libdir} everywhere

that this also points to prefix/bin on Windows, so replaced that as well in https://github.com/JuliaPackaging/Yggdrasil/pull/252/commits/903042ccfcce20fe358e41220c876c097fdb459b

Reply:
That's correct: https://juliapackaging.github.io/BinaryBuilder.jl/dev/build_tips/#Automatic-environment-variables-1

Reply:
Ok great. I see the docs have improved, I should re-read them :)